packages/core-components/src/components/checkbox/checkbox.tsx

```diff
@@ -44,6 +44,9 @@ export class CheckboxComponent {
   /** Whether or not the checkbox is rendered with error styles. Defaults to false. */
   @Prop() invalid?: boolean = false;
 
+  /** @internal whether the parent checkbox-group is disabled. */
+  @Prop() groupDisabled = false;
```

Comment:
Should be optional right ?

Reply:
It isn't reflected and internal and by setting a default we don't run the risk of it being undefined per default.